persistence-cassandra/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/cassandra/CassandraPersistenceComponents.scala

```diff
@@ -21,6 +26,19 @@ trait WriteSideCassandraPersistenceComponents extends WriteSidePersistenceCompon
   override lazy val persistentEntityRegistry: PersistentEntityRegistry =
     new CassandraPersistentEntityRegistry(actorSystem)
 
+  def serviceLocator: Option[ServiceLocator]
```

Comment:
That is the obvious solution and I also think we should try to end up there. I can't find the reason for why we did this optional dance. It must have been something because I would not have implemented that guice magic for fun. I guess we will notice when we run things for real in dev and prod mode (ConductR plays a role here as well).